extensions/dicom-pdf/src/id.js

```diff
@@ -0,0 +1,9 @@
+const Name = 'dicom-pdf';
+const id = `org.ohif.${Name}`;
+
+export default id;
+
+const SOPClassHandlerId = `${id}.sopClassHandlerModule.${Name}`;
```

Comment:
we don't need to have the module name here. It is handled inside the ExtentionManager

Reply:
same for viewportModuleId, the module itself doesn't need an Id, the components it provides should have a name. `deafult` extension has a lot of these you can take a look

Reply:
You can resolve this